web/js/MapUI/Components/GranuleHover/GranuleHover.js

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { getActiveGranuleFootPrints } from '../../../modules/layers/selectors';
+import { GRANULE_HOVERED, GRANULE_HOVER_UPDATE } from '../../../util/constants';
+import util from '../../../util/util';
+
+const { events } = util;
+
+const GranuleHover = (props) => {
+  const {
+    granuleFootprints,
+    state,
+    ui,
+  } = props;
+
+  const [granuleHover, setGranuleHover] = useState({});
+  const [granuleHoverUpdate, setGranuleHoverUpdate] = useState({});
+
+  const onGranuleHover = (platform, date, update) => {
```

Comment:
Do we need "update" as an argument?

Reply:
The only reason I left this parameter there is because this is a function for granules and I'm not sure if maybe it was originally put in there for a reason that may not have been built or utilized yet. I think I will be working with the granules more so I will keep an eye on this one in the future.